src/shared/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "googleapis-common",
+  "version": "0.0.1",
```

Comment:
0.1.0 is much better as 0.0.1 prevents any patch updates (effectively pins the version).
```
> semver.satisfies("0.0.1", "^0.0.1")
true
> semver.satisfies("0.0.2", "^0.0.1")
false
```